src/framework/ui/topNavigationBar/topNavigationBar.component.tsx

```diff
@@ -12,6 +18,50 @@ import {
 } from '@kitten/theme';
 import { Props as ActionProps } from './topNavigationBarAction.component';
 
+/**
+ * The `TopNavigationBar` component is a component that work like AppBar component.
+ * Can be used for navigation for example.
+ *
+ * @extends React.Component
+ *
+ * @property {string} title - Determines the title of the component.
+ *
+ * @property {string} subtitle - Determines the subtitle of the component.
+ *
+ * @property {string} appearance - Determines the appearance of the component.
+ * Can be 'default' | 'titleCentered'.
+ * By default appearance is 'default'.
+ *
+ * @property {React.ReactElement<ActionProps>} leftControl - Determines the left control
+ * of the component.
+ *
+ * @property {React.ReactElement<ActionProps>[]} rightControls - Determines the right controls
```

Comment:
btw, we have a bug with this. Try not sending anything to this prop (it is optional)

Reply:
temporary [fix](https://github.com/akveo/react-native-ui-kitten/pull/324/commits/36f03c8aa0622741bf7b14191ad0fcb0d92abc5b#diff-f9ba568c31f91420551aceefaa097bbbR88)